client/packages/programs/src/JsonForms/common/components/Number.tsx

```diff
@@ -14,17 +20,42 @@ import {
 import { Box } from '@mui/system';
 import { FormLabel } from '@mui/material';
 
-export const numberTester = rankWith(3, schemaTypeIs('number'));
+export const numberTester = rankWith(
+  5,
+  or(schemaTypeIs('number'), uiTypeIs('Number'))
+);
```

Comment:
Allows the "Number" ui component to be used, even if the schema requires a string value